### PR TITLE
Added numpy dependency into requirements.txt

### PR DESCRIPTION
**Pull request by pravee42**
#952 Fix

Added **numPy==1.22.3** into requirements.txt file

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 requests>=2.12.5
 tqdm>=4.59.0
 ruamel.yaml<=0.16.6
-tabulate
\ No newline at end of file
+tabulate
+numpy==1.22.3
```
Comment (reiase) [inline requirements.txt:5]: suggest not including a version, just `numpy` should work for most people.

**Review by reiase — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (pravee42) [reply requirements.txt:5]: Ok will change that

**Comment by pravee42** [comment]
also, change this update  in **Pypi** website also

**Comment by fzliu** [comment]
Looks good. I'll approve it once CI completes.

**Comment by fzliu** [comment]
Looks good. I'll approve it once CI completes.

**Comment by pravee42** [comment]
> Looks good. I'll approve it once CI completes.

thanks,

also, change this update on the Pypi website also then only this will be fixed and the user will able to use an update

**Review by reiase — APPROVED** [review]

**Comment by fzliu** [comment]
@pravee42 Merged - thanks for your contribution. We'll update this in PyPi momentarily.

**Comment by pravee42** [comment]
> @pravee42 Merged - thanks for your contribution. We'll update this in PyPi momentarily.

yeah sure do this ASAP

**Pull request merged**